.sauce/testcafe.yml

```
@@ -1,18 +1,37 @@
 apiVersion: v1alpha
-metadata:
-  name: Testing TestCafe Support
-  tags:
-    - e2e
-    - release team
-    - other tag
-  build: Release $CI_COMMIT_SHORT_SHA
-files:
-  - ./tests/e2e/testcafe/example.test.js
-suites:
-  - name: "saucy test"
-    match: ".*.(spec|test).[jt]s$"
-image:
-  base: saucelabs/stt-testcafe-node
-  version: latest
+kind: testcafe
 sauce:
   region: us-west-1
+  concurrency: 1 # Controls how many suites are executed at the same time (sauce test env only).
+  metadata:
+    name: Testing Testcafe Support
+    tags:
+      - e2e
+      - release team
+      - other tag
+    build: Release $CI_COMMIT_SHORT_SHA
+suites:
+  - name: "saucy test"
+    browser: "firefox" # Although Testcafe supports triggering one test in multiple browsers, it'd better here to split them into every suite to indicate each suite has its own test point.
```

The code was updated, the but config wasn't ℹ️ 
```suggestion
    browserName: "firefox" # Although Testcafe supports triggering one test in multiple browsers, a suite is only meant to run against a single browser.
```

---

updated :)